lib/supply/uploader.rb

```diff
@@ -89,5 +97,26 @@ def client
     def metadata_path
       Supply.config[:metadata_path]
     end
+
+    def skip_upload_apk
+      @skip_upload_apk = Supply.config[:skip_upload_apk]
```

Comment:
These helper methods are a little bit overkill. It would simplify the code to be like:
```
upload_binary unless Supply.config[:skip_upload_apk]
```